samples/bluetooth/rpc_host/prj.conf

```diff
@@ -16,3 +16,5 @@ CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 CONFIG_NRF_RPC_THREAD_STACK_SIZE=4096
+
+CONFIG_NCS_CHILD_IMAGE_SAMPLE_EMPTY_APP_CORE=y
```

Comment:
This in incorrect as the rpc_host sample is typically used as a child image, i.e. the application core triggers its build through a multi-image build. This is also probably why the CI failed.

Reply:
maybe this PR is simply just exposing an existing flaw in our system, but to me it appears wrong to build the `rpc_host` for the netcore and then `empty_app` on the app core.

What is supposed to talk to the `rpc_host` in this case ?

Either we should not build `rpc_host` stand alone for the nRF53 netcore, but only test this sample indirectly through other bluetooth samples.
Or we should ensure a sane sample is actually built for the nRF53 appcore when building `rpc_host` as stand alone.
One option could be to use `peripheral_lbs` as the image for the appcore in this case.

Reply:
the README actually invites the reader to try it with the zephyr beacon sample, maybe we should test it with that instead. But isn't that a bit out of this PR's scope ? I can make another PR to do that once this one is merged.

Reply:
@grochu I reverted the changes, could you resolve the conversation so we can merge ?